.env

```diff
@@ -1,2 +1,2 @@
-ANCHOR_WALLET="/Users/{user}/.config/solana/id.json"
+ANCHOR_WALLET="/Users/chungquantin/snf-1.json"
```

Comment:
could we please revert this back to this : **ANCHOR_WALLET="/Users/{user}/.config/solana/id.json"**